### PR TITLE
add emergency exec copy

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**

### CODE REVIEW

File: governance/votes/Executive vote - March 3, 2023 [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Template - [Executive Vote] Emergency Parameter Changes - March 11, 2023
+summary: Emergency parameter changes to multiple collateral types including Debt Ceiling, Target Borrow Rate, and PSM tin and tout changes.
+date: 2023-03-11T00:00:00.000Z
+address: "$spell_address"
+
+---
+# [Executive Proposal] mergency Parameter Changes - March 11, 2023
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 3, 2023:8]:
```suggestion
# [Executive Proposal] Emergency Parameter Changes - March 11, 2023
```

File: governance/votes/Executive vote - March 3, 2023 [hunk 2]
```diff
@@ -0,0 +1,97 @@
+---
+title: Template - [Executive Vote] Emergency Parameter Changes - March 11, 2023
+summary: Emergency parameter changes to multiple collateral types including Debt Ceiling, Target Borrow Rate, and PSM tin and tout changes.
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 3, 2023:3]:
```suggestion
summary: Emergency parameter changes to multiple collateral types including Debt Ceiling, Target Borrow Rate, Target Available Debt, and PSM tin and tout changes.
```

File: governance/votes/Executive vote - March 3, 2023 [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+- Reduce the GUNIV3DAIUSDC1-A `line` by 100 million DAI from 100 million DAI to **0 DAI**.
+- Reduce the GUNIV3DAIUSDC2-A `line` by 100 million DAI from 100 million DAI to **0 DAI**.
+
+### PSM Parameter Changes
+
+As per this [post](https://forum.makerdao.com/t/emergency-proposal-risk-and-governance-parameter-changes-11-march-2023/20125) from the Risk Core Unit (RISK-001), the following emergency parameter changes will be made, if this executive proposal passes:
+
+#### [Maximum Debt Ceiling (`line`)](https://manual.makerdao.com/module-index/module-dciam#maximum-debt-ceiling-line) Change
+
+- Increase the PSM-USDP-A `line` by 550 million DAI to **1 billion DAI**.
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 3, 2023:51]:
```suggestion
- Increase the PSM-USDP-A `line` by 550 million DAI from 450 million DAI to **1 billion DAI**.
```

File: governance/votes/Executive vote - March 3, 2023 [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+#### [Target Available Debt (`gap`)](https://manual.makerdao.com/module-index/module-dciam#target-available-debt-gap) Changes
+
+- Reduce the PSM-USDC-A `gap` by 700 million DAI from 950 million DAI to **250 million DAI**.
+- Reduce the PSM-GUSD-A `gap` by 40 million DAI from 50 million DAI to **10 million DAI**.
+- Increase the PSM-USDP-A `gap` by 200 million DAI from 50 million DAI to **250 million DAI**.
+
+### DAI Direct Deposit Module (D3M) [Target Borrow Rate (`bar`)](https://manual.makerdao.com/module-index/module-dai-direct-deposit#target-borrow-rate-bar) Changes
+
+**Note that the proposed changes to the D3M `bar` parameters are not subject to the GSM Pause Delay due to the presence of a MOM contract. Therefore, these changes will be actioned when the executive proposal is scheduled**.
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 3, 2023:70]:
```suggestion
**Note that the proposed changes to the D3M `bar` parameters are not subject to the GSM Pause Delay due to the presence of a [MOM contract](https://manual.makerdao.com/governance/verification/gsm-exceptions#direct-deposit-breaker). Therefore, these changes will be actioned when the executive proposal is scheduled**.
```

File: governance/votes/Executive vote - March 3, 2023 [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+As per this [post](https://forum.makerdao.com/t/emergency-proposal-risk-and-governance-parameter-changes-11-march-2023/20125) from the Risk Core Unit (RISK-001), the following emergency parameter changes will be made, if this executive proposal passes:
+
+- Reduce the DIRECT-AAVEV2-DAI `bar` by 2% from 2% to **0%**.
+- Reduce the DIRECT-COMPV2-DAI `bar` by 2% from 2% to **0%**.
+
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 3, 2023:76]: Not sure if it's Makerburn that's incorrect but it says the Aave D3M bar is already 0% and Compound D3M bar is 1.98%. Links: https://makerburn.com/#/collateral/direct_aavev2_dai/params and https://makerburn.com/#/collateral/direct_compv2_dai/params

File: governance/votes/Executive vote - March 3, 2023 [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+### [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) Changes
```
Comment (astronautthis) [inline governance/votes/Executive vote - March 3, 2023:77]:
```suggestion
### [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay) Change
```

File: governance/votes/Executive vote - March 3, 2023 (hunk 5 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - March 3, 2023:76]: it's a makerburn issue

**Pull request merged**